frontend/src/app/modules/authorization/login-form/login-form.component.html

```diff
@@ -0,0 +1,17 @@
+<form action="/login" method="POST" class="login-inp">
+    <span class="p-float-label">
+        <input id="float-input-email" type="text" name="email" pInputText [(ngModel)]= "user.email">
+        <label for="float-input-email">Email</label>
+    </span>
+    <span class="p-float-label">
+        <p-password id="float-input-pass" name="password" [(ngModel)]= "user.password" [style]="{'width':'100%'}" [inputStyle]="{'width':'100%'}" [feedback]="false" [toggleMask]="true"></p-password>
+        <label for="float-input-pass">Password</label>
+    </span>
+    <span class="p-field-checkbox">
+        <p-checkbox class="remember-check" name="rememberMe" [(ngModel)]="rememberMe" [binary]="true" inputId="binary"></p-checkbox>
+        <label for="binary">Remember me</label>
+    </span>
+    <span class="submit-button">
+        <p-button label="Submit" [style]="{'width':'100%'}"></p-button>
```

Comment:
Please don't use inline styles

Reply:
We can`t add styles to p-button in another way, because it`s styles are in the library component